lib/streamlit/elements/write.py

```diff
@@ -181,7 +181,9 @@ def flush_buffer():
             # Order matters!
             if isinstance(arg, str):
                 string_buffer.append(arg)
-            elif type_util.is_snowpark_data_object(arg):
+            elif type_util.is_snowpark_data_object(arg) or type_util.is_type(
```

Comment:
nit: maybe we also add a utils method for pyspark: `is_pyspark_data_object` or `is_pyspark_dataframe`. In that case we don't have to use the internal `_PYSPARK_DF_TYPE_STR` constant here.

Reply:
Yes, good point, it's done and made code more readable 👍